content/release-10-15-0/announcements-10-15-0.md

```diff
@@ -89,16 +89,11 @@ If your microservice is using the deprecated API version 1 and is deployed to a
 
 #### Implemented
 
-##### Analytics Builder - renaming of frgment property name
+##### Analytics Builder - removal of frgment property
 
 The **Measurement Input** block sends the property name that is used for a fragment as part of its output value.
-In previous versions, this property name was `frgment`. This has now been changed to `fragment`.
-Currently, both property names `frgment` and `fragment` are supported.
-`frgment`, however, is now deprecated and will be removed in a future release.
-Therefore it is recommended that you now change `frgment` to `fragment` in all of your blocks
+With release 10.14, this property name was changed from `frgment` to `fragment`, and `frgment` was deprecated. 
```

Comment:
We usually add a link to the original announcement here. For example: "As announced in the 10.14 release notes..."

Reply:
Ok, thanks. I've added a link to the announcement in 10.14 and reworded the entire 10.15 announcement.